docker-monolith/infra/ansible/roles/reddit_monolith/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+env: local
+docker_image: fallgeratoor/otus-reddit:1.0
```

Comment:
it is better to use `reddit_monolith` prefix because `docker_image` is too generic and can be accidentally redefined somewhere else